README.md

```diff
@@ -68,11 +68,32 @@ Getting Started
 
 *if running into weird errors first try `$ rbenv rehash` and restart your terminal*
 
+If you get this error installing the postgres gem and you installed the Posgres.app (and not via brew):
```

Comment:
Yes! I get this error every time and have to search for this fix. Thanks for documenting it!

Reply:
The new analysis looks like this:

<img width="977" alt="screen shot 2019-02-18 at 4 36 05 pm" src="https://user-images.githubusercontent.com/14797300/52981330-5d9d1080-339c-11e9-9772-3856b9bffcaf.png">

Everything adds up correctly, which is great! But we still get (1m) at the end because the rounded sum of segment times and the rounded sum of in-aid times doesn't quite reconcile. I'm wondering if the best solution is to go to one-second resolution...what do you think?